src/component.py

```diff
@@ -47,21 +48,28 @@ def run(self) -> None:
         salt = params.get(KEY_SALT, "")
         salt_location = params.get(KEY_SALT_LOCATION, DEFAULT_SALT_LOCATION)
 
-        tables_to_anonymize = params.get(KEY_TABLES)
+        tables_to_anonymize = params.get(KEY_TABLES, {})
 
         # Check if files are in input instead of tables, give warning in the case of files
         self.check_for_files()
         # Move files from data/in/files to data/out/files
         self.move_files()
 
+        columns_anon_everywhere = params.get(KEY_COLUMNS_TO_ANONYMIZE_EVERYWHERE)
```

Comment:
Mne se tohle teda moc nelibi. je to krkolomny. Co se stane kdyz udelam tohle?:
```
 "parameters": {
    "method": "MD5",
    "#salt" : "salt_string_here",
    "salt_location" : "append",
    "columns_to_anonymize_everywhere": ["Email"],
    "tables_to_encrypt": {
      "test.csv": [
        "Email",
        "Name"
      ]
    }
  }
```
Ja bych tam radsi umoznil wildcard, stejne jako delame u rename columns processoru. Takze pro pokryti tohodle usecase by to udelalo:
```
"tables_to_encrypt": {
      "test.csv": ["email"]}
```

Reply:
Proc to vlastne delame jaky je usecase? Tusim ze to jde od Davida P. zejo? Neni to jenom proto, ze neumi pustit debug job a najit spravny nazev tabulky? Proto si tam chce radsi pausalne encryptovat vsechny sloupce s danym nazvem. Nerekl bych, ze to je dobrej duvod pro to nebyt explicitni v tom co chci anonymizovat.

Reply:
> Mne se tohle teda moc nelibi. je to krkolomny. Co se stane kdyz udelam tohle?:
> 
> ```
>  "parameters": {
>     "method": "MD5",
>     "#salt" : "salt_string_here",
>     "salt_location" : "append",
>     "columns_to_anonymize_everywhere": ["Email"],
>     "tables_to_encrypt": {
>       "test.csv": [
>         "Email",
>         "Name"
>       ]
>     }
>   }
> ```
> 
> Ja bych tam radsi umoznil wildcard, stejne jako delame u rename columns processoru. Takze pro pokryti tohodle usecase by to udelalo:
> 
> ```
> "tables_to_encrypt": {
>       "test.csv": ["email"]}
> ```

encrypt everywhere funguje pouze pokud nejsou specifikovány tables_to_encrypt, wildcard by byla ale také možnost

> Proc to vlastne delame jaky je usecase? Tusim ze to jde od Davida P. zejo? Neni to jenom proto, ze neumi pustit debug job a najit spravny nazev tabulky? Proto si tam chce radsi pausalne encryptovat vsechny sloupce s danym nazvem. Nerekl bych, ze to je dobrej duvod pro to nebyt explicitni v tom co chci anonymizovat.

ano, od Davida, tedy teď jsi mě znejistil, předpokládal jsem, že potřeba anonymizovat daný sloupec ve všech tabulkách plyne z potřeby klienta, doptám se